docs/releases/2.0.txt

```diff
@@ -226,6 +226,11 @@ Models
   number of rows fetched by the Python database client when streaming results
   from the database. For databases that don't support server-side cursors, it
   controls the number of results Django fetches from the database adapter.
+Aggregations
+~~~~~~~~~~~~
+
+* Built in aggregates now accept a `filter` parameter that can be used to
```

Comment:
Double \`\` around filter. You might want to mention it's using the SQL 2003 FILTER clause when the backend supports it and fallback to `When(Case())`?